HLTrigger/Configuration/python/HLT_75e33/modules/hltHgcalSoALayerClustersProducer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+from ..psets.hgcal_reco_constants_cfi import HGCAL_reco_constants as HGCAL_reco_constants
+
+hltHgcalSoALayerClustersProducer = cms.EDProducer("HGCalSoALayerClustersProducer@alpaka",
+    alpaka = cms.untracked.PSet(
+        backend = cms.untracked.string('')
+    ),
+    hgcalRecHitsLayerClustersSoA = cms.InputTag("hltHgcalSoARecHitsLayerClustersProducer"),
+    hgcalRecHitsSoA = cms.InputTag("hltHgcalSoARecHitsProducer"),
+    positionDeltaRho2 = cms.double(1.69),
+    thresholdW0 = cms.double(2.9)
+)
```

Comment:
Why the above import of HGCAL_reco_constants which are then not used in this file?

Reply:
I guess a cut&paste left over. Thanks for spotting this.